### ISSUE TITLE
Feature Request: Datetime Fulfilled Column for Market Orders

### ISSUE THREAD

**Issue by alexanderdarino** [GoldenGnu/jeveassets#65]
For orders that have been fulfilled prior to expiration, I'd love for there to be a column indicating the datetime in which the order was completely (as opposed to partially) fulfilled.

Practical applications: determining recently fulfilled buy/sell orders so you can post a new one for the same item (as opposed to manually going through all the items to see if you have an order already up). I already do this with expired queries ("which orders expired in the past week?"), but it is not presently possible with date fulfilled. 

I don't know if the "datetime fulfilled" is available via ESI, but if it isn't the information can be inferred from transactions: the datetime fulfilled is the datetime of the last transaction for that market order. (In the unlikely event it comes up, it is very easy to disambiguate multiple orders for the same item and price point)

**Comment by GoldenGnu** [comment]
The problem is matching each transaction to a specific order. There is no 100% reliable way to figure out what order a transaction was made to/from. You can guess, as I said, but, you can have situations where it's impossible to know. Also, it is a lot of work writing all the code ensuring you're getting right as often as possible. Whereas my suggested solution is a simple change.

**Comment by GoldenGnu** [comment]
Yes, I can agree with that. Using the best available data sounds like a good plan.
So, feature request accepted. But, without an ETA. It will be done at some point, but, I can not promise when.

**Comment by alexanderdarino** [comment]
Where do I donate ISK to push this up the list 😂 ?

**Comment by GoldenGnu** [comment]
> Where do I donate ISK to push this up the list
Someone with more free time than me :D 
> You're involved in the CSM, right
The CSM member is Steve, not me.
> Would it make sense to petition to include order_id in the transaction data on ESI?
It turns out there was already a ESI issue for adding order_id.
However, it was close, so, after asking in #esi I have re-opened the issue: https://github.com/esi/esi-issues/issues/82
